packages/fast-components-styles-msft/src/pivot/index.ts

```diff
@@ -0,0 +1,106 @@
+import { DesignSystem, withDesignSystemDefaults } from "../design-system";
+import {
+    ComponentStyles,
+    ComponentStyleSheet,
+    CSSRules,
+} from "@microsoft/fast-jss-manager";
+import {
+    applyFocusVisible,
+    applyLocalizedProperty,
+    Direction,
+    toPx,
+} from "@microsoft/fast-jss-utilities";
+import { ensureBrandNormal, hoverContrast } from "../utilities/colors";
+import outlinePattern from "../patterns/outline";
+import { PivotClassNameContract } from "@microsoft/fast-components-class-name-contracts-msft";
+
+const styles: ComponentStyles<PivotClassNameContract, DesignSystem> = (
+    config: DesignSystem
+): ComponentStyleSheet<PivotClassNameContract, DesignSystem> => {
+    const designSystem: DesignSystem = withDesignSystemDefaults(config);
+    const direction: Direction = designSystem.direction;
+
+    return {
+        pivot: {
+            position: "relative",
+            color: designSystem.foregroundColor,
+            overflow: "hidden",
+        },
+        pivot_tabList: {
+            display: "flex",
+            boxSizing: "border-box",
+        },
+        pivot_tab: {
+            minHeight: "32px",
+            padding: "0 12px",
+            fontSize: "14px",
+            whiteSpace: "nowrap",
+            display: "flex",
+            border: "2px solid",

```

Comment:
Why not just `border: "2px solid transparent", instead of bother `border` and `borderColor`?

Reply:
Because we are updating the borderColor in the focus state and you can run into weirdness if you use short hand and then target an individual property.